src/main.c

```diff
@@ -128,14 +139,17 @@ void handleApdu(volatile unsigned int *flags, volatile unsigned int *tx, int rx)
             switch (e & 0xF000) {
```

Comment:
should G_called_from_swap be reset to false in the CATCH block above? (CATCH(ApduReplySdkExceptionIoReset))

Reply:
it re-throws but I'm not sure if exception thrown from one catch block will be re-caught in the next

Reply:
is there a need to reset it actually, if we return from the app next time it is called it will set the variable to a proper value

Reply:
If it throws the app quit and there is no problem
What we want to avoid is the loop starting again with `G_called_from_swap == true` while we have seen an error